README.md

```diff
@@ -64,7 +64,7 @@ Newsboat can be compiled.
     UPDATE doc/newsboat.asciidoc IF YOU CHANGE THIS LIST
 -->
 - GNU Make 4.0 or newer
-- C++ compiler that accepts `-std=c++17` option: GCC 5.0 or newer, or Clang 5 or
+- C++ compiler that accepts `-std=c++17` option: GCC 7.0 or newer, or Clang 5 or
```

Comment:
The "accepts `-std=c++17`" wording was chosen because it's really all we need: simply accepting the option is enough, even if there is no actual C++17 support. Now we do require C++17, so let's change this to:

```suggestion
- C++17 compiler: GCC 7.0 or newer, or Clang 5 or
```

Also, please update same paragraph in doc/newsboat.asciidoc as well.

Reply:
Good point, done